samples/apps/android/LwcTestApp/app/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        applicationId "com.salesforce.mobiletooling.lwctestapp"
+        minSdkVersion 23
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
I'm curious which version of Android Studio you generated this project from? A number of these artifacts seems like they're from an older version (things like JDK 7 references). Could we start with an Android Studio 4.x template?

Reply:
I just updated my AS to 4.1.1 this morning and tried creating a new Kotlin project!

Reply:
Ah weird. I wonder where that jdk7 type stuff is coming from? New projects would typically be explicitly targeting JVM 1.8, e.g. in [app/build.gradle](https://github.com/forcedotcom/lwc-dev-mobile/pull/81/files#diff-7904bd2129b804d123e2c84f3b698efac7dc24c9031e8ff381d65132c003efe9), I'd expect to see:
```gradle
kotlinOptions {
    jvmTarget = '1.8'
}
```
🤔

Reply:
I don't know what was the problem. I rebooted my machine, then created an empty Kotlin project and started recoding it from the scratch. Seems to have fixed the problem this time!

Reply:
Ah yes, that looks more like what I would expect. Thanks!